<launch>
 
    <!-- Start the PathGenerator node -->
    <node name="path_generator_node" pkg="assignment6" type="PathGenerator.py" output="screen">
      <param name="plan_topic" type="string" value="/planner_node/car_plan" />
      <param name="source_topic" type="string" value="/initialpose" />
      <param name="target_topic" type="string" value="/move_base_simple/goal" />
      <param name="good_file_name" type="string" value="good_waypoints-1.csv" />
      <param name="bad_file_name" type="string" value="bad_waypoints-1.csv" />
      <param name="start_file_name" type="string" value="start-1.csv" />
    </node>

</launch>
